node/nodeTesting_test.go

```diff
@@ -417,3 +419,141 @@ func TestGenerateAndSendBulkTransactions_ShouldWork(t *testing.T) {
 	assert.Equal(t, noOfTx, len(recoveredTransactions))
 	mutRecoveredTransactions.RUnlock()
 }
+
+func TestNode_SendBulkTransactionsMultiShardTxsShouldBeMappedCorrectly(t *testing.T) {
```

Comment:
Might also add an integration test based on the model found on integrationTests/multiShard/transaction/interceptedResolvedBulkTx_test.go , tests 1 and 2

Reply:
done